tools/cell_census_builder/__main__.py

```diff
@@ -139,6 +139,8 @@ def build(
     if args.consolidate:
         consolidate(args, top_level_collection.uri)
 
+    add_git_commit_sha(top_level_collection)
```

Comment:
Let's move this to `create_top_level_collections()`, to keep it with the other top-level metadata setters.  Or move all of those here, if you think that's better (which seems reasonable to me).